### PR TITLE
BumpTechnologies.BumpTop version 2.5.6268

### PR DESCRIPTION
**Pull request by Brandon621**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16402)

### CODE REVIEW

File: manifests/b/BumpTechnologies/BumpTop/2.5.6268/BumpTechnologies.BumpTop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BumpTechnologies.BumpTop
+PackageVersion: 2.5.6268
+PackageLocale: en-US
+Publisher: Bump Technologies
```
Comment (ItzLevvie) [inline manifests/b/BumpTechnologies/BumpTop/2.5.6268/BumpTechnologies.BumpTop.locale.en-US.yaml:6]:
```suggestion
Publisher: Bump Technologies, Inc.
```
image

File: manifests/b/BumpTechnologies/BumpTop/2.5.6268/BumpTechnologies.BumpTop.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BumpTechnologies.BumpTop
+PackageVersion: 2.5.6268
+PackageLocale: en-US
+Publisher: Bump Technologies
+PackageName: BumpTop
+License: Apache License 2.0
+ShortDescription: BumpTop Setup
```
Comment (ItzLevvie) [inline manifests/b/BumpTechnologies/BumpTop/2.5.6268/BumpTechnologies.BumpTop.locale.en-US.yaml:9]:
```suggestion
ShortDescription: BumpTop is a 3D desktop user interface inspired by real desks using physics, multi-touch gestures to drive towards a more expressive, human vision for computing.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm receiving this prompt on my machine:
image

Other than the changes I've made, and this prompt, everything else looks good to me:
image

**Comment by Brandon621** [comment]
> I'm receiving this prompt on my machine:
> image

You just need .NET 3.5

**Review by ItzLevvie — APPROVED** [review]
It works fine without .NET Framework; only the DirectX install which the application installed was needed for the application to install without the user's interaction with mouse and keyboard:
image

image

**Pull request merged**